src/components/ProductCard/ProductCard.scss

```
@@ -0,0 +1,90 @@
+.productCard {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  .cardContainer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+
+    width: 420px;
+    height: 423px;
+    margin: 25px 0 20px;
+
+    .cardThumbnail {
+      width: 400px;
+      height: 281px;
+
+      img {
+        width: 100%;
+        height: 100%;
+      }
+    }
+
+    .discriptionContainer {
+      display: flex;
+      justify-content: flex-start;
+      flex-direction: column;
+
+      width: 100%;
+      padding: 14px 12px 0;
```

```suggestion
      display: flex;
      justify-content: flex-start;
      flex-direction: column;
      width: 100%;
      padding: 14px 12px 0;
```
가독성을 위해 한칸 띄우는 건 선택자 사이만 해당합니다
한 선택자 안의 스타일은 붙여서 사용해주세요~

---

넵 알겠습니다!